apps/core/cli/populate/index.ts

```diff
@@ -81,6 +81,7 @@ async function createDatabases({ whichModels }: PopulateOptions) {
       }
     });
     await Promise.all(content);
+    console.log('populating database...');
```

Comment:
why did you not use a looger here instead of console.log?

Reply:
the cli does not know and it's not going to know about anything from the project since it`s a tech debit.